### DISCUSSION TITLE
Multiple settings profiles

### DISCUSSION THREAD

**Discussion by userrand** [monkeytypegame/monkeytype#4545] [Ideas]
I have a set of settings I use for practicing speed and another for practicing accuracy and it is quite tedious to change multiple settings each time I want to practice something else. It would be nice if there was a way to have a speed profile that could load settings I defined for speed and an accuracy profile with another set of settings.

**Comment by Miodec** [comment]
Have you tried using presets?

**Reply by userrand** [reply]
Is that only available when you register your account? I am not sure where to find presets

**Reply by Miodec** [reply]
Yes, only for registered accounts. Its in the settings.
